libs/dh/admin/feature-user-management-new/src/lib/details/details.component.html

```diff
@@ -74,10 +74,14 @@ <h2>{{ user.firstName + " " + user.lastName }}</h2>
             <dh-user-master-data [user]="user" />
           </watt-tab>
           <watt-tab [label]="t('roles.tabLabel')">
-            <dh-user-roles [user]="user" [expanded]="false" />
+            @defer {
+              <dh-user-roles [id]="user.id" [expanded]="false" />
+            }
           </watt-tab>
           <watt-tab *dhPermissionRequired="['fas']" [label]="t('history.tabLabel')">
-            <dh-user-audit-logs [user]="user" />
+            @defer {
+              <dh-user-audit-logs [id]="user.id" />
+            }
```

Comment:
praise: nice change. The `@defer` is still a bit of an underused feature. It can help with perf quite a lot